server/core/settings.py

```diff
@@ -12,6 +12,7 @@ class Settings(BaseSettings, env_file=".env"):
     LOG_NAME: str = "server"
     LOG_ACCESS_NAME: str = "server.access"
 
+    lf_use_data_dir: bool = False
     lf_project_dir: str | None = None
```

Comment:
Should we remove `lf_project_dir`?

Reply:
I was thinking of leaving it for now, since there might be a use if you want to explicitly set the project dir a startup for testing. But we may want to get rid of it at some point.